.github/workflows/ci.yaml

```diff
@@ -167,3 +167,18 @@ jobs:
           file: ${{ matrix.connector }}/Dockerfile
           push: true # See 'if' above
           tags: ghcr.io/estuary/${{ matrix.connector }}:dev
+
+      - name: Push ${{ matrix.connector }} image with version tag
+        if: ${{ github.event_name == 'push' }}
+        run: |
+          export IMAGE_VERSION=$(grep VERSION ${{ matrix.connector }}/Dockerfile | cut -d = -f 2)
+          version_exists=$(docker manifest inspect ghcr.io/estuary/${{ matrix.connector}}:$IMAGE_VERSION > /dev/null ; echo $?)
```

Comment:
Does this check have a race condition if there are multiple merges to master when one of them bumps the version number? Or is there something preventing that?

Reply:
@willdonnelly that is a good point. I will disable concurrency on main branch following [this](https://github.com/estuary/flow/blob/master/.github/workflows/main.yml#L15-L17)